xinference/core/restful_api.py

```diff
@@ -266,6 +277,20 @@ def serve(self):
             methods=["POST"],
             response_model=ChatCompletion,
         )
+
+        # for custom models
+        self._router.add_api_route(
+            "/v2/models/register_model", self.register_model, methods=["POST"]
```

Comment:
Consider using `/v1/model_registrations` for better adherence to RESTful API design. Here are the suggested endpoints:

Register an LLM: POST /v1/model_registrations/llm
Unregister an LLM: DELETE /v1/model_registrations/llm
List LLM registrations: GET /v1/model_registrations/llm
Get an LLM registration: GET /v1/model_registrations/llm/${model_name}

Reply:
udpated in the latest commit